indico/web/client/styles/base/_pages.scss

```diff
@@ -46,33 +46,40 @@
 
     .title {
       display: flex;
-      align-items: flex-start;
-      padding-top: 0.4rem;
-      min-height: 2rem;
+      align-items: center;
+      flex-wrap: wrap;
+      gap: 0.5em;
+      margin-bottom: 1em;
+      padding: 0.5em 3em;
+
       border-bottom: 1px solid $separator-color;
-      margin-bottom: 1rem;
 
       > .text {
         flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+        gap: 0.5em;
 
         > .title-with-actions {
-          h2,
-          h3 {
-            display: inline;
-            margin-right: 0;
+          display: flex;
+          gap: 0.5em;
+          align-items: center;
+          justify-content: space-between;
+
+          h1 {
+            // FIXME: font attributes of headings should not be defined globally, fix in the base styles later
```

Comment:
Not sure if related, but there is some minor layout breakage (as in, it looked better before). Probably just some missing styling...

Before:
![image](https://github.com/indico/indico/assets/179599/709b83cd-4872-40af-b362-df7b7a7f9ca4)
![image](https://github.com/indico/indico/assets/179599/513ff6fb-d964-4e54-8b5e-7d6263a4f971)

After:
![image](https://github.com/indico/indico/assets/179599/958f74bf-7e0b-4c6c-a463-d4ab57e2d144)
![image](https://github.com/indico/indico/assets/179599/e0dc533c-73e8-4b8d-9404-4db2a7200cf9)

Reply:
Hmmmm, how do I interpret this?

```
{% block title %}
    {% if subcategories %}
        {% trans %}Categories{% endtrans %}
        {# If the category also contains events, a second title will be displayed further down the page. #}
    {% elif events.total %}
        {% trans %}Events{% endtrans %}
    {% else %}
        {% trans %}Content{% endtrans %}
    {% endif %}
{% endblock %}
```

Reply:
I'm confused that seemingly unrelated things are being switched here.

Reply:
Ok, so this is a bit more serious than we though. I modified the template markup in `base.html` to match the usage on the search page **semantically**. This makes the markup on the management page completely non-semantic. (I don't know how I missed that it was used on the management page, which is totally my bad.)

Reuse of the markup should be focused on the semantics and not the visual appearance, but sadly that was not the case. Also, this base template is used in a lot of places, so if all these usages follow the same pattern... it'll be virtually impossible to address heading levels without making a fairly intrusive change that touches just about all pages where this template is used.

I'll need to scrap this PR because with the current template usage, I cannot possibly make this fix work. I'm not quite sure what the best way to attack this is, because I'm not yet familiar with all the pages in the app. I'm inclined to leave this for some later date when I'm more familiar with everything.